src/main/java/com/imsweb/datagenerator/naaccr/NaaccrDataGenerator.java

```diff
@@ -85,7 +88,7 @@ public abstract class NaaccrDataGenerator implements DataGenerator {
      * Constructor
      * @param useMaidenNameField if true, the name rule will use the 'nameMaiden' field, otherwise it will use the 'nameBirthSurname' field
      */
-    public NaaccrDataGenerator(boolean useMaidenNameField) {
+    protected NaaccrDataGenerator(boolean useMaidenNameField) {
```

Comment:
Why did you make this constructor protected? How do you know it's not called for some other projects?

Reply:
Since I have a new code cleanup issue for this project, I'll just fix all instances in that issue.

Reply:
I think that is a good idea.